### PR TITLE
[ID] Add Auto and Easy (mod) articles

### PR DESCRIPTION
**Pull request by hikikuman**
Add Indonesian translation for [Auto](https://osu.ppy.sh/wiki/en/Game_modifier/Auto) and [Easy](https://osu.ppy.sh/wiki/en/Game_modifier/Easy) articles (game modified).

### CODE REVIEW

File: wiki/Game_modifier/Easy/id.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Tidak seperti mode permainan lainnya, penggunaan mod Easy di osu!taiko dapat mengurangi persyaratan skor note untuk mendapatkan health, serta membuat life bar terisi lebih cepat sebagai pengganti dari dua "nyawa" tambahan karena tidak mungkin fail di tengah-tengah permainan.
+
+### osu!catch
+
+Di [osu!catch](/wiki/Game_mode/osu!catch), efeknya sama seperti di osu!standard dengan beberapa kesejajaran dan penyesuaian: semua buah jatuh lebih lambat (AR meningkat), kameranya "diperbesar" (CS meningkat), dua "nyawa" tambahan diberikan.
+
+Meski hanya menggunakan satu sumbu, ukuran buah yang lebih besar dan kecepatan jatuh yang lebih lambat dapat menyebabkan buah-buah tersebut menggumpal, sehingga timbul masalah yang sama pada kesulitan di osu!standard.
+
+<!-- image needed here -->
```
Comment (AncuL001) [inline wiki/Game_modifier/Easy/id.md:61]:
```suggestion
![Gameplay EZ pada catch](img/EZ-comparison-catch.jpg "Perbandingan antara permainan biasa (kiri) dan permainan dengan mod Easy (kanan) pada osu!catch")
```

**Review by AncuL001 — CHANGES_REQUESTED** [review]
lol you actually used an old image, it's probably obsolete by now

File: wiki/Game_modifier/Auto/id.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Di [osu!taiko](/wiki/Game_mode/osu!taiko), mod Auto akan mencoba permainan yang sempurna, tetapi tidak selalu mendapatkan grade SS. Mod ini terkadang akan melewatkan satu detakan pada drumrolls yang tidak dimulai dengan satu atau setengah ketukan dan tidak akan selalu bisa menyelesaikan dendens (spinners) pada map dengan overall difficulty yang sangat tinggi.
+
+### osu!catch
+
+Di [osu!catch](/wiki/Game_mode/osu!catch), mod Auto benar-benar akan melampaui kesempurnaan dan menggunakan kecepatan yang tak terbatas untuk menangkap *semua buah pisang* pada sebuah spinner, meskipun prestasi seperti itu mustahil bagi seorang manusia.
+
+### osu!mania
+
+Di [osu!mania](/wiki/Game_mode/osu!mania), mod Auto akan mencoba permainan yang sempurna, tetapi mungkin akan miss pada note yang tertumpuk (mis., penekanan tombol ditengah-tengah sebuah note panjang).
```
[the remark this answers is not in the document]
Comment (hikikuman) [reply wiki/Game_modifier/Auto/id.md]: terdapat > terjadi

File: wiki/Game_modifier/Auto/id.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+*Berbeda konteks dengan [Autopilot (mod)](/wiki/Game_modifier/Autopilot).*
+
+## Tentang
+
+- Singkatan: AT atau AO
+- Jenis: Spesial
+- Pengali Skor: 1.00x
+- Tombol shortcut: `V`
+- Deskripsi: `Tonton permainan sempurna yang berjalan secara otomatis`<!-- yes, that is written correctly -->
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:27]:
```suggestion
- Mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/Auto/id.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+*Perhatian: Penggunaan mod Auto akan menyebabkan permainan menjadi unranked dan tidak akan tersimpan di papan peringkat lokal.*
+
+Mod **Auto** adalah sebuah [game modifier](/wiki/Game_modifier) yang memungkinkan pemain untuk menonton sebuah permainan yang sempurna dari beatmap yang dipilih.
+
+Fungsi mod ini sama seperti sebuah [tayangan ulang](/wiki/Replay), dengan opsi pemutaran kecepatan 0.5x/1.0x/2.0x yang dapat digunakan, dan komentar pada beatmap yang dapat dilihat dan dibuat seperti biasa. Mod Auto akan selalu mendapatkan grade SS, dengan pengecualian tertentu dalam beberapa kasus.
+
+### osu!standard
+
+Di [osu!standard](/wiki/Game_mode/osu!), saat bertemu dengan sebuah spinner, mod Auto akan memutar spinner pada kecepatan maksimal yaitu 477 spins per minute (SPM) dengan arah yang berlawanan dengan arah jarum jam.
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:39]:
```suggestion
Di [osu!](/wiki/Game_mode/osu!), saat bertemu dengan sebuah spinner, mod Auto akan memutar spinner pada kecepatan maksimal yaitu 477 spins per minute (SPM) dengan arah yang berlawanan dengan arah jarum jam.
```

File: wiki/Game_modifier/Auto/id.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### osu!mania
+
+Di [osu!mania](/wiki/Game_mode/osu!mania), mod Auto akan mencoba melakukan permainan yang sempurna, tetapi mungkin akan terjadi miss pada note yang tertumpuk (misalnya penekanan tombol di tengah sebuah note panjang).
+
+## Trivia
+
+- Saat menonton sebuah tayangan ulang Auto, nama pengguna yang ditampilkan pada papan peringkat akan menjadi "osu!."
+
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:57]:
```suggestion
[osu!]: /wiki/shared/mode/osu.png "osu!"
```

File: wiki/Game_modifier/Auto/id.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:58]:
```suggestion
[osu!taiko]: /wiki/shared/mode/taiko.png "osu!taiko"
```

File: wiki/Game_modifier/Auto/id.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:59]:
```suggestion
[osu!catch]: /wiki/shared/mode/catch.png "osu!catch"
```

File: wiki/Game_modifier/Auto/id.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
```
Comment (DoryanR) [inline wiki/Game_modifier/Auto/id.md:60]:
```suggestion
[osu!mania]: /wiki/shared/mode/mania.png "osu!mania"
```

File: wiki/Game_modifier/Easy/id.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+
+## Tentang
+
+- Singkatan: EZ
+- Jenis: Pengurang Kesulitan
+- Pengali Skor: 0.50x
+- Tombol shortcut: `Q`
+- Deskripsi: 
+  - ![][o!s]: `Mengurangi tingkat kesulitan keseluruhan - lingkaran membesar, HP drain yang lebih kecil, timing window yang lebih toleran, dan lainnya!`
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:25]:
```suggestion
  - ![][osu!]: `Mengurangi tingkat kesulitan keseluruhan - lingkaran membesar, HP drain yang lebih kecil, timing window yang lebih toleran, dan lainnya!`
```

File: wiki/Game_modifier/Easy/id.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+  - ![][o!t]: `Mengurangi tingkat kesulitan map secara umum - not bergerak lebih lambat, serta membutuhkan tingkat akurasi yang lebih rendah.`
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:26]:
```suggestion
  - ![][osu!taiko]: `Mengurangi tingkat kesulitan map secara umum - not bergerak lebih lambat, serta membutuhkan tingkat akurasi yang lebih rendah.`
```

File: wiki/Game_modifier/Easy/id.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+  - ![][o!c]: `Mengurangi tingkat kesulitan keseluruhan - lingkaran membesar, HP drain yang lebih kecil, timing window yang lebih toleran, dan lainnya!`
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:27]:
```suggestion
  - ![][osu!catch]: `Mengurangi tingkat kesulitan keseluruhan - lingkaran membesar, HP drain yang lebih kecil, timing window yang lebih toleran, dan lainnya!`
```

File: wiki/Game_modifier/Easy/id.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+  - ![][o!m]: `Mengurangi tingkat kesulitan map secara umum - HP drain yang lebih kecil, dan membutuhkan akurasi yang lebih kecil.`
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:28]:
```suggestion
  - ![][osu!mania]: `Mengurangi tingkat kesulitan map secara umum - HP drain yang lebih kecil, dan membutuhkan akurasi yang lebih kecil.`
```

File: wiki/Game_modifier/Easy/id.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+- Mode permainan yang cocok: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:29]:
```suggestion
- Mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```

File: wiki/Game_modifier/Easy/id.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+## Deskripsi
+
+Mod **Easy** adalah sebuah [game modifier](/wiki/Game_modifier) yang mencoba meringankan kesulitan pada sebuah [beatmap](/wiki/Beatmaps). Cara kerja mod ini adalah dengan mengurangi setengah dari semua pengaturan kesulitan dari beatmap yang dipilih.
+
+Di semua [mode permainan](/wiki/Game_mode) kecuali [osu!taiko](/wiki/Game_mode/osu!taiko), mod Easy akan memberikan dua "nyawa" tambahan kepada pemain saat sedang bermain jika [life bar](/wiki/Glossary#life-bar) mereka turun ke 0 (habis). Saat kasus tersebut terjadi, permainan akan berhenti sejenak untuk mengisi ulang life bar ke sekitar 80% dan menghapus satu "nyawa" dari jumlah nyawa pemain. Tidak akan ada suara yang diputar selama efek ini.
+
+Pemain akan diberi tanda berupa suara ready (`readysound.wav`) saat sedang pengisian ulang life bar. Setelah life bar selesai diisi ulang, pemain akan diberi tanda lagi berupa suara go (`gosound.wav`). Setelah itu, beatmap akan kembali dilanjutkan dan memperbolehkan pemain untuk melanjutkan bermain. Jika pemain tidak mempunyai sisa nyawa dan life bar mereka turun ke 0 (habis), permainan akan fail seperti biasa.
+
+### osu!standard
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:39]:
```suggestion
### osu!
```

File: wiki/Game_modifier/Easy/id.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+Di [osu!standard](wiki/Game_Modes/osu!), mod Easy menurunkan setengah dari [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size), [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty), dan [HP drain](/wiki/Beatmap_Editor/Song_Setup#hp-drain-rate) (HP).
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:41]:
```suggestion
Di [osu!](wiki/Game_Modes/osu!), mod Easy menurunkan setengah dari [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size), [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty), dan [HP drain](/wiki/Beatmap_Editor/Song_Setup#hp-drain-rate) (HP).
```

File: wiki/Game_modifier/Easy/id.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+![Gameplay EZ pada standard](img/EZ-comparison-standard.jpg "Perbandingan antara permainan biasa (kiri) dan permainan dengan mod Easy (kanan) pada osu!standard")
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:43]:
```suggestion
![Gameplay EZ pada standard](img/EZ-comparison-standard.jpg "Perbandingan antara permainan biasa (kiri) dan permainan dengan mod Easy (kanan) pada osu!")
```

File: wiki/Game_modifier/Easy/id.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+Di [osu!taiko](/wiki/Game_mode/osu!taiko), mod Easy menurunkan setengah dari [slider velocity](/wiki/Glossary#slider-velocity), overall difficulty (OD) dan HP drain.
+
+*Catatan: Jumlah hit yang dibutuhkan untuk menyelesaikan sebuah [spinner](/wiki/Hit_object/Spinner) dikurangi sesuai dengan nilai OD.*
+
+Tidak seperti mode permainan lainnya, penggunaan mod Easy di osu!taiko dapat mengurangi persyaratan skor note untuk mendapatkan health, serta membuat life bar terisi lebih cepat sebagai pengganti dari dua "nyawa" tambahan karena tidak mungkin fail di tengah-tengah permainan.
+
+### osu!catch
+
+Di [osu!catch](/wiki/Game_mode/osu!catch), efeknya sama seperti di osu!standard dengan beberapa kesejajaran dan penyesuaian: semua buah jatuh lebih lambat (AR meningkat), kameranya "diperbesar" (CS meningkat), dua "nyawa" tambahan diberikan.
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:59]:
```suggestion
Di [osu!catch](/wiki/Game_mode/osu!catch), efeknya sama seperti di osu! dengan beberapa kesejajaran dan penyesuaian: semua buah jatuh lebih lambat (AR meningkat), kameranya "diperbesar" (CS meningkat), dua "nyawa" tambahan diberikan.
```

File: wiki/Game_modifier/Easy/id.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+
+Meski hanya menggunakan satu sumbu, ukuran buah yang lebih besar dan kecepatan jatuh yang lebih lambat dapat menyebabkan buah-buah tersebut menggumpal, sehingga timbul masalah yang sama pada kesulitan di osu!standard.
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:61]:
```suggestion
Meski hanya menggunakan satu sumbu, ukuran buah yang lebih besar dan kecepatan jatuh yang lebih lambat dapat menyebabkan buah-buah tersebut menggumpal, sehingga timbul masalah yang sama pada kesulitan di osu!.
```

File: wiki/Game_modifier/Easy/id.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+
+Di  [osu!mania](/wiki/Game_mode/osu!mania), efeknya sama seperti di osu!taiko, kecuali pemberian dua nyawa tambahan yang diberikan kepada pemain, dan slider velocity serta jumlah key bawaan akan tetap sama.
+
+## Trivia
+
+- Saat menonton sebuah tayangan ulang di mana nyawa tambahan tersebut digunakan, tidak akan ada animasi pengisian ulang health dan tayangan ulang akan tetap berlanjut seolah-olah pemain sedang bermain menggunakan mod [No Fail](/wiki/Game_modifier/No_fail).
+- Di mode [Multi](/wiki/Multi), efek "nyawa" tidak akan bekerja di mode tim [Tag Co-op atau Tag Team Vs.](/wiki/Multi#tag-coop-tag-team-vs).
+- Mod Easy akan membatalkan mod [Hard Rock](/wiki/Game_modifier/Hard_Rock) secara otomatis, dan sebaliknya.
+
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:75]:
```suggestion
[osu!]: /wiki/shared/mode/osu.png "osu!"
```

File: wiki/Game_modifier/Easy/id.md [hunk 20]
```diff
@@ -0,0 +1,78 @@
...
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:76]:
```suggestion
[osu!taiko]: /wiki/shared/mode/taiko.png "osu!taiko"
```

File: wiki/Game_modifier/Easy/id.md [hunk 21]
```diff
@@ -0,0 +1,78 @@
...
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:77]:
```suggestion
[osu!catch]: /wiki/shared/mode/catch.png "osu!catch"
```

File: wiki/Game_modifier/Easy/id.md [hunk 22]
```diff
@@ -0,0 +1,78 @@
...
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
```
Comment (DoryanR) [inline wiki/Game_modifier/Easy/id.md:78]:
```suggestion
[osu!mania]: /wiki/shared/mode/mania.png "osu!mania"
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
In line with the new standards

File: wiki/Game_modifier/Auto/id.md (hunk 3 above)
Comment (TicClick) [reply wiki/Game_modifier/Auto/id.md:27]: these are the URL references (a full link for each is usually at the bottom of the article). they are invisible, intentionally made short, and their names do not matter

Comment (TicClick) [reply wiki/Game_modifier/Auto/id.md:27]: the same goes for other PRs where you left similar suggestions

Comment (mousesalt) [reply wiki/Game_modifier/Auto/id.md:27]: i think this was reference of
image
that recently changed on FA showcase article

**Review by wowcake — APPROVED** [review]

**Review by wowcake — APPROVED** [review]
should be good to go 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**